internal/templates/element/base.go

```diff
@@ -32,6 +32,28 @@ const {{.ElementName}}Limbs = {{.NbWords}}
 // {{.ElementName}}Bits number bits needed to represent {{.ElementName}}
 const {{.ElementName}}Bits = {{.NbBits}}
 
+// ToBytes converts z into a slice of byte
+// the resulting slice is seen as a big endian integer
+func (z *{{.ElementName}}) ToBytes() []byte {
```

Comment:
Should be named `Bytes`, like in `math/big`: https://golang.org/pkg/math/big/#Int.Bytes 

Instead of "the resulting slice is seen as[..]" (documentation string), something like `Bytes returns the regular (non montgomery) value of z as a big-endian byte slice.`